docs/websockets.md

```diff
@@ -65,14 +65,14 @@ This section will specifically discuss deployment in the context of websockets a
 | `nginx`     | listens on ports 80/443, handles HTTPS proxying, serves static assets, routes requests for `daphne` and `uwsgi` |
 | `uwsgi`      | listens on port 8050, handles API requests |
 | `daphne`      | listens on port 8051, handles websocket requests |
-| `wsbroadcast`   | no listening port, forwards all group messages to all cluster nodes |
+| `wsrelay`   | no listening port, forwards all group messages to all cluster nodes |
```

Comment:
> forwards all group messages to all cluster nodes

Request clarification here - would it be more accurate to say that it forwards all group messages to "all web nodes"

Perhaps it would be good to add something to the doc about how something qualifies as a web node, considering that it is different from the `Instance` registration mechanism.

I'm not even sure about the "all group messages" part. I thought I remember something about the web nodes declaring what groups they have clients subscribed to, so only messages in certain groups/channels need to be sent to certain web nodes.

But "no listening port" still sounds right!

Reply:
I'm going to resolve this comment to get it off my screen. I will be rewriting (or heavily modifying) `docs/websockets.md` before this PR is ever ready for review. :)